docs/3.api/2.composables/use-nuxt-data.md

```diff
@@ -52,28 +72,34 @@ const { data } = await useAsyncData('todos', () => $fetch('/api/todos'))
 ```vue [components/NewTodo.vue]
 <script setup lang="ts">
 const newTodo = ref('')
-const previousTodos = ref([])
+let previousTodos = []
 
 // Access to the cached value of useAsyncData in todos.vue
 const { data: todos } = useNuxtData('todos')
 
-const { data } = await useFetch('/api/addTodo', {
-  method: 'post',
-  body: {
-    todo: newTodo.value
-  },
-  onRequest () {
-    previousTodos.value = todos.value // Store the previously cached value to restore if fetch fails.
-
-    todos.value.push(newTodo.value) // Optimistically update the todos.
-  },
-  onRequestError () {
-    todos.value = previousTodos.value // Rollback the data if the request failed.
-  },
-  async onResponse () {
-    await refreshNuxtData('todos') // Invalidate todos in the background if the request succeeded.
-  }
-})
+const addTodo = async () => {
```

Comment:
As a not, I prefer using the function declaration when possible for code clarity when scanning what are variables vs functions:

```diff
- const addTodo = async () => {
+ async function addTodo () {
```

cc @Mini-ghost 

Beautiful PR btw ❤️